lib/core-integration/src/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -324,6 +323,7 @@ spec = describe "SHELLEY_STAKE_POOLS" $ do
                   "withdrawal": "self"
                 }|]
 
+        waitForNextEpoch ctx
```

Comment:
:tada:

Reply:
I have run this test 35 times so far and it's not failed yet.

Reply:
Max confidence is reached at 42 times.

Reply:
It failed with the other problem (new error message now) on the 82nd test.

```
[82 2021-01-08 18:31:30] Failures:
[82 2021-01-08 18:31:30] 
[82 2021-01-08 18:31:30]   src/Test/Integration/Scenario/API/Shelley/StakePools.hs:208:5: 
[82 2021-01-08 18:31:30]   1) API Specifications, SHELLEY_STAKE_POOLS, STAKE_POOLS_JOIN_01rewards - Can join a pool, earn rewards and collect them
[82 2021-01-08 18:31:30]        uncaught exception: RequestException
[82 2021-01-08 18:31:30]        DecodeFailure "{\"code\":\"network_query_failed\",\"message\":\"Unable to query the ledger at the moment. This error has been logged. Trying again in a bit might work.\"}"
[82 2021-01-08 18:31:30] 
[82 2021-01-08 18:31:30]   To rerun use: --match "/API Specifications/SHELLEY_STAKE_POOLS/STAKE_POOLS_JOIN_01rewards - Can join a pool, earn rewards and collect them/"
[82 2021-01-08 18:31:30] 
[82 2021-01-08 18:31:30] Randomized with seed 962601967
[82 2021-01-08 18:31:30] 
[82 2021-01-08 18:31:30] Finished in 28.4153 seconds
[82 2021-01-08 18:31:30] 1 example, 1 failure
[82 2021-01-08 18:31:30] 
real	108m6.389s
user	52m39.964s
sys	5m12.888s
```